_studio/mfx_lib/encode_hw/h265/include/hevce_ddi_main.h

```diff
@@ -112,3 +112,13 @@ typedef struct tagENCODE_CAPS_HEVC
     UINT     reserved32bits3;
 } ENCODE_CAPS_HEVC;
 
+//We cannot modify original struct, but we need to add new checks.
+struct MFX_ENCODE_CAPS_HEVC
+{
+    ENCODE_CAPS_HEVC ddi_caps;
```

Comment:
In my opinion, it would be better if you extend structure ENCODE_CAPS_HEVC, not include it in another one: struct MFX_ENCODE_CAPS_HEVC : ENCODE_CAPS_HEVC {...} In that case, you wouldn't need to change all existing caps with duplicated naming: m_caps.ddi_caps

Reply:
We can't. It's from DDI.

Reply:
@otsirule Yes, but with inclusion it will be clear where our custom part of caps, and where ddi part.
Also we discussed about this in [changes for h264](https://github.com/Intel-Media-SDK/MediaSDK/pull/1432), and I think we must stick to one code style.

Reply:
@dmitryermilov , we can. As Anton mentioned, you've discussed this in another pull request. 
I can't approve these changes, because find them excessive and it's difficult to work with such code. But I won't block it.